Gemfile

```diff
@@ -36,7 +36,7 @@ gem 'newrelic_rpm'
 gem 'pg'
 gem 'phonelib'
 gem 'premailer-rails', '>= 1.11.1'
-gem 'proofer', github: '18F/identity-proofer-gem', tag: 'v2.7.1'
+gem 'proofer', github: '18F/identity-proofer-gem', ref: '01c75dcdc25792f62ed12018a72516e338768dfb'
```

Comment:
TODO: make this v0.8.0 once https://github.com/18F/identity-proofer-gem/pull/49 is merged